docs/UserGuide.adoc

```diff
@@ -377,57 +412,83 @@ TODO: ADD DETAILS ONCE IMPLEMENTED
 [[FilterStudent]]
 ==== Filter `filter`
 
-TODO: ADD DETAILS ONCE IMPLEMENTED
+Filter Student returns you all the students according to the given keywords below.
+
+You can filter students in a class using these two ways:
+
+****
+1. module code and group code `m/` `g/`
+2. module code `m/`
+****
+
+1. Using both module code and group code returns you all the students specified by the group code and module code.
+
+Format: `student filter [m/MOD_CODE] g/GROUP_CODE`
+
+Examples:
+
+* ` student filter m/cs2103t g/g06`
+Filters all students in module `CS2103T`, under group `G06`.
+
+2. Using just the module code will return you all the students from the first group of module code shown in UI.
+
+Format: `student filter m/MOD_CODE`
+
+Examples:
+
+* `student filter m/cs2103t`
+Filters all the students in module `CS2103T`, first group of CS2103T that is currently showing in UI.
+
+[NOTE]
+Keywords are case-insensitive. eg. `cs2103t` will match `CS2103T`
+
 
 === Session View
 
 [[AddSession]]
 ==== Adding a Session `session add`
 
-TODO - UPDATE
+Adds a new session. The new undone session will be shown under Session view.
 
-Adds a new type of session for an existing module group in the program.
-The `CATEGORY_UID` for this command is the SESSION_TYPE code.
+Format: `session add m/MOD_CODE [s/START] [e/END] [d/DATE] [w/WEEKS] [t/SESSION_TYPE] [n/NOTES]`
 
-Format: `add c/session s/START e/END d/DATE [-r] m/MOD_CODE [t/SESSION_TYPE] [n/NOTES]`
+[TIP]
+If you want to create a session at this point of time, you can leave the start time `s/START`, end time `e/END` and date `d/DATE` out. A session of current date and time will be automatically generated.
 
 [NOTE]
 ====
 [horizontal]
-`START` `END` `DATE`:: see Common Parameters
-
-`[-r]`::  makes the session recur at the same time every week
+`START` `END` `DATE` `WEEK`:: see Common Parameters
 
 `MOD_CODE` `SESSION_TYPE`:: see Common Parameters
 
 `NOTES`:: notes for this session (eg: weekly tutorials or prep sessions)
```

Comment:
If we're just asking them to refer to common parameters, we can leave this part out. You could add NOTES to the common parameter list and get rid of this section all together.